src/pad.c

```diff
@@ -1,4 +1,7 @@
+/* -*- mode: c; c-file-style: "bsd"; indent-tabs-mode: t; -*- */
+
 /*
+ * Copyright (c) 2016 Luka Novsak <lnovsak@gmail.com>
```

Comment:
Copyrights shouldn't be updated on file changes. Git already tracks who contributed to the file.

It's perfectly legitimate to take credit for all the amazing work you've done here, so maybe add an `AUTHORS` file at the root directly and add both our names into that?

Or I'd be okay with removing the entire copyright notice altogether if you prefer.

Appending names would get messy over time and also at this point doesn't reflect the tens of people who contributed to those files already.